src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -121,7 +121,7 @@ function FilterBarUI(props: Props) {
           withTitle
           panelPaddingSize="none"
           ownFocus={true}
-          initialFocus=".filterEditor__hiddenItem"
+          initialFocus='[data-test-subj="filterFieldSuggestionList"] [data-test-subj="comboBoxSearchInput"]'
```

Comment:
Aren't `data-test-subj` are meant for functional tests ? Can we provide another class ?

Reply:
Hi, you suggest adding a class? sure, I wouldn't mind

Reply:
It could be class or a unique id attribute, I am fine either way as long as this will take care It should be good.